examples/archive.html

```diff
@@ -105,10 +105,22 @@ <h4>Share via email</h4>
   <div class="offcanvas offcanvas-end" data-bs-backdrop="false" data-bs-keyboard="false" tabindex="1" id="offcanvasRight" aria-labelledby="offcanvasRightLabel">
       <div class="offcanvas-header">
         <button type="button" id="restoreWelcomeModalBtn" class="btn btn-primary btn-md" data-bs-dismiss="offcanvas" aria-label="Close" title="Import images from collection URL">Import</button>
-       
-        <button id="downloadJSON" type="button" class="btn btn-sm" aria-label="Save map" title="Save map in JSON format" >
-          <i class="fa fa-save fa-" style="font-size:24px" ></i>
+       <div class="dropdown">
+  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" title="Save/Download Map">
+    <i class="fa fa-save fa-" style="font-size:24px" ></i>
+  </button>
+  <ul class="dropdown-menu">
+    <li  title="Save map locally in JSON format"> 
+      <button id="downloadJSON" type="button" class="btn btn-sm dropdown-item" aria-label="Save map" >
+          Download Map
+        </button></li>
+    <li title="Save map in browser in JSON format"> 
```

Comment:
What if we added one more option, which uses alert() to provide a bit more text about the limitations and uses of each option? What might we say to let people understand the difference?

Reply:
Hmm, that sounds reasonable, something like an [info icon](https://fontawesome.com/v5/icons/info-circle?style=solid&s=solid&f=classic)?

I believe we could give a write-up like:
`Saving your map in the browser means you can continue knitting when you return to mapknitter.org`

I believe the share modal succinct info on what the downloaded JSON file could be used for.

@jywarren

Reply:
Sounds great but instead of mapknitter.org, let's say "this page in your current browser" to really get specific about what's required to recover the state. Sound good?

Reply:
Sounds good!

Maybe we'd worry about recovering the state in another PR, but what would that flow be like?

Reply:
Well, one option could be to have this save in localstorage button open the history modal, and do it all in one PR, or do the history modal first, so there's something to open when we implement this button? That modal could have space to better explain things.

Reply:
Hmm, I thought we resolved to place a button [here](https://github.com/publiclab/Leaflet.DistortableImage/pull/1225#issuecomment-1421128530)
or should they both open the modal?

Also, then which button triggers the actual save to local storage

Reply:
I know it sounds strange that the second item in the Save button would then open the History modal, when there's another button for that. But consider this flow:

<img width="700" alt="image" src="https://user-images.githubusercontent.com/24359/219174352-f97e4d11-98ac-42ee-9b29-78099e959670.png">

I think it makes sense to have the two forms of saving be together. But I also think that having the Saved Maps modal be the place where you can save or recover "in browser" maps makes sense. I can see how it's a little strange that `Download` immediately does something while `Save in browser` takes you somewhere else. But it also links saving and recovering together nicely. 

What do you think?

Reply:
I think it's not a bad idea, a tad confusing at the first glance but not bad.
I'll begin work on it, should it be done in this PR or another?

Reply:
I think it's OK to do here, if you like? After all, it also makes sense that we should introduce the ability to recover a map before we introduce the ability to save one, otherwise people may save maps but not be able to recover them, you know?

Reply:
If you think of an improved way to do it, I'm happy to discuss! Thanks Malik!

Reply:
> I think it's OK to do here, if you like? After all, it also makes sense that we should introduce the ability to recover a map before we introduce the ability to save one, otherwise people may save maps but not be able to recover them, you know?

Yea I totally agree 
I'll get it on here

> If you think of an improved way to do it, I'm happy to discuss! Thanks Malik!

Definitely 
Thank you @jywarren